Added custom Cypress command to create an article. (#13151)
forem/forem: docs/tests/e2e-tests.md

[knapsack-pro-cypress](https://github.com/KnapsackPro/knapsack-pro-cypress) npm
package.

## Cypress Custom commands

[Cypress custom commands](https://docs.cypress.io/api/cypress-api/custom-commands.html)

> allow you do extend the functionality of the E2E testing framework. In the case

Reviewer: ```suggestion
allow you to extend the functionality of the E2E testing framework. In the case
```
I _think_ this is what you meant? :)

> of Forem, we need custom commands. For example, like creating an article.

Reviewer: ```suggestion
of Forem, we need custom commands to create an article, for example.
```

A custom command is prefixed like any Cypress command by `cy.` All custom
commands can be found in the

> https://github.com/forem/forem/blob/master/cypress/support/commands.js file.

Reviewer: ```suggestion
[commands.js](https://github.com/forem/forem/blob/master/cypress/support/commands.js) file.
```

> ### Create Article Custom Command

Reviewer: ```suggestion
### Creating a Custom Article Command
```
